src/test/scala/sigmastate/serialization/TwoArgumentSerializerSpecification.scala

```diff
@@ -15,6 +15,8 @@ class TwoArgumentSerializerSpecification extends TableSerializationSpecification
       (Minus(LongConstant(2), LongConstant(3)), Array[Byte](MinusCode, SLong.typeCode, 4, SLong.typeCode, 6)),
       (Plus(LongConstant(2), LongConstant(3)), Array[Byte](PlusCode, SLong.typeCode, 4, SLong.typeCode, 6)),
       (Multiply(LongConstant(2), LongConstant(3)), Array[Byte](MultiplyCode, SLong.typeCode, 4, SLong.typeCode, 6)),
+      (Min(LongConstant(2), LongConstant(3)), Array[Byte](MinCode, SLong.typeCode, 4, SLong.typeCode, 6)),
+      (Max(LongConstant(2), LongConstant(3)), Array[Byte](MaxCode, SLong.typeCode, 4, SLong.typeCode, 6)),
```

Comment:
@greenhat I think it is better to avoid such magic constants. ZigZag encoding, why not use is here explicitly?

Reply:
Done.